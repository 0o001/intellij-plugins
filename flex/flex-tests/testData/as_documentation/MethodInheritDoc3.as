package {

public class Sub extends Base {

  /**
   * @inheritDoc
   */
  public override function f<caret>oo(p1:String, p2:int) : Boolean {

  }
}
}
