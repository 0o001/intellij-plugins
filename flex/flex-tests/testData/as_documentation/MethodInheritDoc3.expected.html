<div class='definition'><pre>Sub.foo(p1: String,<br>        p2: int): Boolean</pre></div><table class='sections'><tr><td valign='top' class='section'><p>Params:</td><td><p>p1 &ndash;  param_1<p>p2 &ndash;  param_2</table>